website/project/metadata/aea-prereg-2.json

```diff
@@ -0,0 +1,480 @@
+{
+    "name": "AEA_Preregistration",
+    "version": 2,
+    "description": 
+    "pages": [{
+        "id": "page1",
+        "title": "",
+        "contents": [{
+            "id": "general",
+            "type": "section",
+            "contents": [{
+                "required": true,
+                "type": "textfield",
+                "id": "title",
+                "label": "Trial Title",
+                "caption": "caption"
+            }, {
+                "type": "textfield",
+                "id": "registration_number",
+                "label": "AEA RCT Registration Number",
+                "caption": "caption"
+            }, {
+                "type": "textfield",
+                "id": "registration_date",
+                "label": "Registration Date",
+                "caption": "caption"
+            }, {
+                "type": "textfield",
+                "id": "updated",
+                "label": "Last Updated",
+                "caption": "caption"
+            }],
+            "title": "General Information"
+        }, {
+            "type": "section",
+            "canRepeat": true,
+            "id": "location",
+            "contents": [{
+                "type": "select",
+                "required": true,
+                "options": [
```

Comment:
This schema is a reformatting of the v1 schema of the same name.

It's contents are not new. 

If you think the content of the AEA Prereg schema should be updated, that should be addressed outside of this PR.